apps/verticalhome/js/more_apps_navigation.js

```diff
@@ -69,6 +70,11 @@
           this.items[this.selectedElemIndex].element.click();
           e.preventDefault();
           break;
+        case 'c':
+          if(this.personalize_mode){
```

Comment:
this.personalize_mode = document.getElementById('more-apps-screen') always will be true. Probably, you mean checked for enable "personalize mode". May be should be better use speciacl variable like isInPersonalizeMode?